host/src/lib/HelperFunctions.jsx

```diff
@@ -347,39 +343,45 @@ export const checkEqualityWithOtherAnswers = (rawAnswer, normValue, normType, pr
  * @returns {IResponse[]}
  */
 export const buildShortAnswerResponses = (prevShortAnswer, choices, newAnswer, newAnswerTeamName, teamId) => {
+  let correctAnswer = choices.find(choice => choice.isAnswer).text;
+  const correctAnswerRegEx = new RegExp(`\\b${correctAnswer.toLowerCase()}\\b`, 'g');
```

Comment:
we use this Regex to check substrings with the correct answer only.  So if we had the following:

correct: "Paris" 
answer 1: "The answer is Paris"
answer 2: "The answer is" 
answer 3: "the"

answer 1 will match the correct answer while answer 2 and 3 will not match the other answers